package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rabbit-queue",
-  "version": "4.1.2",
+  "version": "4.2.0-beta",
```

Comment:
Is the version the intended one?

Reply:
Will bump the version after this is merged